collector/iis.go

```diff
@@ -1,24 +1,28 @@
+//go:build windows
```

Comment:
We're not mandating Go 1.17 yet, my understanding is that the current syntax is still supported? Any other reason to have double tags?

Reply:
Fixed. Apologies, not sure how that got in there.